tools/linux_build_and_test.sh

```diff
@@ -41,9 +41,14 @@ fi
 cd $(dirname $0)/..
 BASE_DIR=$(pwd)
 
-docker build -t build-image tools/linux
+docker build -t build-linux tools/linux
 docker run -v /var/run/docker.sock:/var/run/docker.sock -v \
```

Comment:
Why is `-v /var/run/docker.sock:/var/run/docker.sock` needed?

Also, as a nit, could you put `-v` to the same line as its argument? Same below.

Reply:
` -v /var/run/docker.sock:/var/run/docker.sock` isn't needed -- I'd had it because it was present in gRPC-node's process for generating pre-compiled binaries; but our script works without this.

Also tried to adjust line breaks to keeps flags and their arguments together.

Reply:
So as FYI, `-v /var/run/docker.sock:/var/run/docker.sock` is only needed if you need to run docker within docker for whatever reason.

Reply:
Thanks!
I knew that was a difference between gRPC-node's workflow and ours, but didn't know it was related to `-v /var/run/docker.sock:/var/run/docker.sock`!